packages/terra-date-input/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added hot keys functionality to month select
```

Comment:
is the shortcut added only for month select ..?

Reply:
No, added for Month, date and year as well. Will update the change log.